pkgs/development/tools/gotools/default.nix

```diff
@@ -2,16 +2,20 @@
 
 buildGoModule rec {
   name = "gotools-unstable-${version}";
-  version = "2019-06-03";
-  rev = "8aaa1484dc108aa23dcf2d4a09371c0c9e280f6b";
+  version = "2019-07-03";
+  rev = "2214986f1668163f6c7315bcfa4a6496af4c3829";
 
   src = fetchgit {
     inherit rev;
     url = "https://go.googlesource.com/tools";
-    sha256 = "0sa41fi38b6pvz7jjr6vqrd152qjvmbcagm1qdxw41vqcdw3ljx3";
+    sha256 = "07k0bgl5w2106ahvyd3lvfd3j02b40h73vmhqnb5mbj3xa8sibyb";
   };
 
-  modSha256 = "0cm7fwb1k5hvbhh86kagzsw5vwgkr6dr7glhbjxg5xaahlhx2w5w";
+  postPatch = ''
```

Comment:
can you please add a comment above `postPatch` explaining why is it removing the postPatch module?

Reply:
Done